### DISCUSSION TITLE
Manufacturer/Name/Serial/Firmware not shown

### DISCUSSION THREAD

**Discussion by Sophist-UK** [nutdotnet/WinNUT-Client#204] [Q&A] [answered]
Manufacturer/Name/Serial/Firmware not shown:
![image](https://github.com/user-attachments/assets/cdb46dd5-2bc7-405e-847c-4e4c30a953b6)

However most of this data is clearly available:
![image](https://github.com/user-attachments/assets/9a584f37-2750-4cd4-b77d-15953a1c9df4)
![image](https://github.com/user-attachments/assets/d50b0875-4776-4396-8760-2851fe45fdbc)
![image](https://github.com/user-attachments/assets/03a314c2-a71c-41bf-b376-c1647caa57ae)

**Comment by Sophist-UK** [comment]
P.S. Other minor issues:

1. Battery 12v is shown even though the UPS does not provide this info.
2. Name is defined in settings but is not in the UPS variables.
3. Model is available but not shown in the main window.

**Comment by gbakeman** [comment]
Hi @Sophist-UK ,

Thanks for taking the time to open this and post all of your observations. I just converted this to a Q&A discussion first so we can get to the bottom of what's going on.

It does definitely seem like there's a bug. Would you please make sure logging is enabled and set to Debug level, make WinNUT reconnect to the NUT server, then drag the log file into a reply box in this window to upload it.

Thanks in advance!

**Reply by Sophist-UK** [reply, marked as answer]
I turned on logging, disconnected and reconnected and got this:
![image](https://github.com/user-attachments/assets/98fac309-9948-4204-b669-6a3b4ea3b998)
{shrug}

I guess this can be closed.

**Reply by Sophist-UK** [reply]
P.S. For next time, where is the log file stored?

**Reply by gbakeman** [reply]
> P.S. For next time, where is the log file stored?

WinNUT stores its data in (your user profile)/AppData/Local or Roaming/NutDotNet/WinNUT Client and further down there will be a version number, then the log file. As a shortcut, there's a button near the logging options that opens an Explorer window to the log file location.

It is very strange that the issue resolved its self. I think I remember someone else commenting that changing a setting affects WinNUT's display in unexpected ways. There's a lot of work I have yet to do in that regard, so stay tuned for improvements. I'll mark your answer here, but feel free to post back if the situation changes.

Thanks again for your efforts here and have a good new year!
